hbase-native-client/CMakeLists.txt

```diff
@@ -226,4 +226,9 @@ INSTALL (
     DIRECTORY ${CMAKE_SOURCE_DIR}/include/
     DESTINATION include/
     FILES_MATCHING PATTERN "*.h*")
+# Install pb-generated headers too
+INSTALL (
+    DIRECTORY ${CMAKE_BINARY_DIR_GEN}
+    DESTINATION include/hbase/if
+    FILES_MATCHING PATTERN "*.h")
```

Comment:
Thanks! If you could change this to "hbase/if/*.h" then I'll merge it. It's trite, but I want what is installed into that directory to be specific to our installation. Granted this won't work on windows ( because of the relative paths I used ) but I think what we have is fine for now.